mbtools/copy_resources_s3.py

```diff
@@ -131,11 +158,13 @@ def add_new_resources_to_s3(bucket, s3_dir, hashes_to_update,
                 "original_filename": os.path.basename(
                     hash_to_filedata_map[hash_key]['path']
                 ),
-                "s3_key": full_keypath
+                "s3_key": full_keypath,
+                "timestamp": datetime.now(timezone.utc).isoformat()
```

Comment:
I think you're doing this because `upload_file` does not return anything. This is probably fine for all intents and purposes, but maybe for our future selves we should comment that, technically, it's not quite "correct". For example, on a file uploaded on a run, subsequent runs will likely show a slightly different timestamp by going through the code path above where we read from the data from `head_object`.

Reply:
Yup, that was the reason why. I have updated the code with a comment.